package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "compressed-size-action",
-  "version": "2.5.0",
+  "version": "3.0.0",
```

Comment:
Is the major version bump here necessary? I'm genuinely asking. I don't know if using `node20` could potentially break anything downstream for any consumers of this action.